autotest/gdrivers/zarr_driver.py

```diff
@@ -5498,7 +5514,9 @@ def test_zarr_driver_rename(format):
     try:
         drv.Create(filename, 1, 1, options=["FORMAT=" + format])
 
-        assert gdal.Open(filename)
+        with gdal.quiet_errors():
```

Comment:
I'm thinking a `gdal.quiet_warnings()` would be a good idea (I'm happy to implement it)

Reply:
If you are up for it, `gdal.quiet_warnings()` seems like a nice addition.

Reply:
`gdal.quiet_warnings()` is now in master.

Reply:
Thanks! Now I will need to work on my git skills for rebasing.

1c404c681301476ff25843e3b7a080a693812d22

Reply:
My usual pattern is (from the feature branch)
```
git fetch upstream master:master
git rebase master
git push -f
```

Reply:
Should be done. Thanks for the git tips.